cmd/batch.go

```diff
@@ -15,10 +16,18 @@ import (
 type BatchPipeliner interface {
 	// afs afero.Afero should be part of struct
 	GetChanges(config core.Config, searchPaths []string) ([]core.Change, error)
-	WriteTemplate(writer io.Writer, template string, pregap bool, templateData interface{}) error
+	WriteTemplate(
+		writer io.Writer,
+		template string,
+		pregapLines int,
+		postgapLines int,
+		templateData interface{},
+	) error
 	WriteFile(
 		writer io.Writer,
 		config core.Config,
+		pregapLines int,
+		postgapLines int,
```

Comment:
Lets rename to beforeNewlines and afterNewlines to match the variable names in other places.
```suggestion
		beforeLines int,
		afterLines int,
		templateData interface{},
	) error
	WriteFile(
		writer io.Writer,
		config core.Config,
		beforeNewlines int,
		afterNewlines int,
```